src/main/resources/application.yml

```
@@ -12,7 +12,7 @@ spring:
         dialect: org.hibernate.dialect.MySQLDialect
         default_batch_fetch_size: 500
     hibernate:
-      ddl-auto: none
+      ddl-auto: update
```

```suggestion
      ddl-auto: none
```
서버 뜰때 ddl 실행하면 신경써아 할게 많아서
개발환경에서는 ddl-auto로 개발하시고
엔티티 변경사항 있을때마다 따로 ddl 실행하겠습니다